app/main/views/sign_in.py

```diff
@@ -153,12 +153,34 @@ def sign_in():
     other_device = current_user.logged_in_elsewhere()
     notify_env = os.getenv("NOTIFY_ENVIRONMENT")
     current_app.logger.info("should render the sign in template")
+
+    # TODO REMOVE THIS INFO ONCE STAGING WORKS WITH LOGIN DOT GOV
+    current_app.logger.info(f"NOTIFY ENV = {notify_env}")
+    current_app.logger.info(
+        f"LOGIN_DOT_GOV_CLIENT_ID={os.getenv('LOGIN_DOT_GOV_CLIENT_ID')}"
+    )
+    current_app.logger.info(
+        f"LOGIN_DOT_GOV_USER_INFO_URL={os.getenv('LOGIN_DOT_GOV_USER_INFO_URL')}"
+    )
+    current_app.logger.info(
+        f"LOGIN_DOT_GOV_ACCESS_TOKEN_URL={os.getenv('LOGIN_DOT_GOV_ACCESS_TOKEN_URL')}"
+    )
+    current_app.logger.info(
+        f"LOGIN_DOT_GOV_LOGOUT_URL={os.getenv('LOGIN_DOT_GOV_LOGOUT_URL')}"
+    )
+    current_app.logger.info(
+        f"LOGIN_DOT_GOV_BASE_LOGOUT_URL={os.getenv('LOGIN_DOT_GOV_BASE_LOGOUT_URL')}"
+    )
+    current_app.logger.info(
+        f"LOGIN_DOT_GOV_SIGNOUT_REDIRECT={os.getenv('LOGIN_DOT_GOV_SIGNOUT_REDIRECT')}"
+    )
+
     return render_template(
         "views/signin.html",
         form=form,
         again=bool(redirect_url),
         other_device=other_device,
-        notify_env_is_dev=bool(notify_env == "development"),
+        notify_env_is_dev_or_staging=bool(notify_env in ["development", "staging"]),
```

Comment:
Would it make sense to change this variable to something like `login_gov_enabled` instead?

```suggestion
        login_gov_enabled=bool(notify_env in ["development", "staging"]),
```

Reply:
I guess so.  The variable is going away once login.gov is available on prod, so it's temporary anyway.